setup.cfg

```diff
@@ -92,7 +92,7 @@ omit =
     tests/*
 
 [coverage:report]
-fail_under = 100
+fail_under = 95
```

Comment:
Our rationale was that 100% coverage was not accurate, since there were many `pragma: no cover` statements in the code. Now that we removed them, we make explicit to everyone that the repo is not at 100% coverage, and we'd need to work to achieve it.

Reply:
Oh, I see. I must have been looking at the commits out of order. Ok, this seems like an ok approach to take, but definitely does introduce more risk of untested code sneaking in rather than being explicitly called out with a `pragma: no cover` . I'm happy to merge this if we're all ok with that. Approving now